sdktests/common_tests_events_custom.go

```diff
@@ -38,7 +37,7 @@ func (c CommonEventTests) CustomEvents(t *ldtest.T) {
 			t.Run(contexts.Description(), func(t *ldtest.T) {
 				context := contexts.NextUniqueContext()
 
-				dataSource := NewSDKDataSource(t, mockld.EmptyServerSDKData())
+				dataSource := NewSDKDataSource(t, nil)
```

Comment:
If you do not specify the data, then it will try to make data for the correct SDK type based on the context.